### PR TITLE
Add entities to Roborock documentation

### PR DESCRIPTION
**Pull request by Lash-L**
## Proposed change

Adds a bit more information about what each entitiy does to the documentation. Open to all suggestions for any kind of restyling.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/roborock.markdown [hunk 1]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
 
 {% include integrations/config_flow.md %}
 
+
+## Entities
+
+Roborock devices have a variety of features that are supported on some devices but not others, only entities that your device supports will be added to your integration
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Roborock devices have a variety of features that are supported on some devices but not on others. Only entities that your device supports will be added to your integration.
```

File: source/_integrations/roborock.markdown [hunk 2]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+### Vacuum
+
+The vacuum entity holds the ability to control most theiongs the vacuum can do, such as start a clean, return to the dock, or set fan speed
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
The vacuum entity holds the ability to control most things the vacuum can do, such as start a clean, return to the dock, or set the fan speed.
```

File: source/_integrations/roborock.markdown [hunk 3]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+### Select
+
+Mop mode - Describes how to mop the floor. On some firmware, it is called 'mop route'
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Mop mode - Describes how to mop the floor. On some firmware, it is called 'mop route'.
```

File: source/_integrations/roborock.markdown [hunk 4]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+Mop intensity - How hard you would like your vacuum to mop
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Mop intensity - How hard you would like your vacuum to mop.
```

File: source/_integrations/roborock.markdown [hunk 5]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+### Sensor
+
+Cleaning area - How much area the Vacuum has cleaned in its current run, or if the vacuum is not currently cleaning, how much area it cleaned during its last run
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Cleaning area - How much area the vacuum has cleaned in its current run.  If the vacuum is not currently cleaning, how much area it has cleaned during its last run.
```

File: source/_integrations/roborock.markdown [hunk 6]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+Cleaning time - How long the Vacuum has been cleaning for, or if the vacuum is not currently cleaning, how long it cleaned for last
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Cleaning time - How long the vacuum has been cleaning for. If the vacuum is not currently cleaning, how long it cleaned for in its last run.
```

File: source/_integrations/roborock.markdown [hunk 7]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+Main brush time left - How much time is left before Roborock recommends you replace your main brush
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Main brush time left - How much time is left before Roborock recommends you replace your main brush.
```

File: source/_integrations/roborock.markdown [hunk 8]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+Side brush time left - How much time is left before Roborock recommends you replace your side brush
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Side brush time left - How much time is left before Roborock recommends you replace your side brush.
```

File: source/_integrations/roborock.markdown [hunk 9]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+Filter time left - How much time is left before Roborock recommends you replace your vacuum's air filter
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Filter time left - How much time is left before Roborock recommends you replace your vacuum's air filter.
```

File: source/_integrations/roborock.markdown [hunk 10]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+Status - The current status of your vacuum, this typically describes the action that is currently being run, i.e. 'spot_cleaning' or 'docking'
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Status - The current status of your vacuum. This typically describes the action that is currently being run. For example, 'spot_cleaning' or 'docking'.
```

File: source/_integrations/roborock.markdown [hunk 11]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+Total cleaning time - The lifetime cleaning duration of your vacuum
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Total cleaning time - The lifetime cleaning duration of your vacuum.
```

File: source/_integrations/roborock.markdown [hunk 12]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+Total cleaning area - The lifetime clenaing area of your vacuum
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Total cleaning area - The lifetime cleaning area of your vacuum.
```

File: source/_integrations/roborock.markdown [hunk 13]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+Vacuum error - The current error with your vacuum, if one exists
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Vacuum error - The current error with your vacuum, if there is one.
```

File: source/_integrations/roborock.markdown [hunk 14]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+### Time
+
+Do not disturb begin - When Do not disturb is enabled, do not run the vacuum or speak after this point
+
+Do not disturb end - When Do not disturb is enabled, do not run the vacuum or speak before this point
+
+### Switch
+
+Child lock - This disables the buttons on the vacuum from doing anything when hit
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Child lock - This disables the buttons on the vacuum. Nothing happens when the buttons are pushed.
```

File: source/_integrations/roborock.markdown [hunk 15]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+Status indicator light - This is the LED on the top of your vacuum that is colored based off of the status of your vacuum
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Status indicator light - This is the LED on the top of your vacuum. The color changes depending on the status of your vacuum.
```

File: source/_integrations/roborock.markdown [hunk 16]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+Total cleaning area - The lifetime clenaing area of your vacuum
+
+Vacuum error - The current error with your vacuum, if one exists
+
+### Time
+
+Do not disturb begin - When Do not disturb is enabled, do not run the vacuum or speak after this point
+
+Do not disturb end - When Do not disturb is enabled, do not run the vacuum or speak before this point
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Do not disturb end - When _Do not disturb_ is enabled, the vacuum does not run or speak before this point.
```

File: source/_integrations/roborock.markdown [hunk 17]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+Total cleaning time - The lifetime cleaning duration of your vacuum
+
+Total cleaning area - The lifetime clenaing area of your vacuum
+
+Vacuum error - The current error with your vacuum, if one exists
+
+### Time
+
+Do not disturb begin - When Do not disturb is enabled, do not run the vacuum or speak after this point
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Do not disturb begin - When _Do not disturb_ is enabled, the vacuum does not run or speak after this point.
```

File: source/_integrations/roborock.markdown [hunk 18]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+Do not disturb end - When Do not disturb is enabled, do not run the vacuum or speak before this point
+
+### Switch
+
+Child lock - This disables the buttons on the vacuum from doing anything when hit
+
+Status indicator light - This is the LED on the top of your vacuum that is colored based off of the status of your vacuum
+
+Do not disturb - This enables Do not disturb during the do not distrub time frame you have set in the app or on the time entity
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Do not disturb - This enables _Do not disturb_ during the time frame you have set in the app or on the time entity. When _Do not disturb_ is enabled, the vacuum does not run or speak.
```

File: source/_integrations/roborock.markdown [hunk 19]
```diff
@@ -34,6 +34,60 @@ Once you log in with your Roborock account, the integration will automatically d
...
+
+### Number
+
+Volume - This allows you to control the volume of the robot's voice, i.e. when it states "Starting cleaning". This allows you to set the volume to 0%, while the app limits it to 20%
```
Comment (c0ffeeca7) [inline source/_integrations/roborock.markdown]:
```suggestion
Volume - This allows you to control the volume of the robot's voice. For example, when it states "Starting cleaning". This allows you to set the volume to 0%, while the app limits it to 20%.
```

**Review by c0ffeeca7 — CHANGES_REQUESTED** [review]
Some small style-related tweaks.

**Comment by Lash-L** [comment]
> Some small style-related tweaks.

I appreciate you looking at it so closely and catching all my typos! Thank you!

**Review by c0ffeeca7 — APPROVED** [review]
LGTM, @Lash-L :+1:

**Pull request merged**